eland/ml/ml_model.py

```diff
@@ -254,6 +261,7 @@ def import_model(
         classification_weights: Optional[List[float]] = None,
         es_if_exists: Optional[str] = None,
         es_compress_model_definition: bool = True,
+        inference_config: Optional[Dict[str, Dict[str, Any]]] = None,
```

Comment:
This is more a semantic change but I suspect we should use `Mapping` instead of `Dict` here.